CONTRIBUTING.md

```diff
@@ -1,187 +1,71 @@
-1. Fixing bugs is extraordinarily helpful and requires the least familiarity with ActiveModelSerializers.
-  Look for issues labeled [**Needs Bug Verification**](https://github.com/rails-api/active_model_serializers/labels/Needs%20Bug%20Verification) and [**Bug**](https://github.com/rails-api/active_model_serializers/labels/bug).
-1. Adding or fixing documentation is also fantastic!
-
-To fetch & test the library for development, do:
-
-1. Fork the repository ( https://github.com/rails-api/active_model_serializers/fork )
-1. `git clone https://github.com/{whoami}/active_model_serializers.git`
-1. `cd active_model_serializers`
-1. `bundle`
-  - To test against a particular rails version-- 4.0 is usually the most buggy-- set then
-      RAILS_VERSION environment variable as described in the [.travis.yml](.travis.yml).
-      e.g. `export RAILS_VERSION=4.0`.
-1. Create your PR branch (`git checkout -b my-helpful-pr`)
-1. Write tests for your feature, or regression tests highlighting a bug.
-  This is important so ActiveModelSerializers doesn't break it in a future version unintentionally.
-1. Write the feature itself, or fix your bug
-1. `bundle exec rake`
-1. Commit your changes (`git commit -am 'Add some feature'`)
-  - Use well-described, small (atomic) commits.
-1. Push to the branch (`git push origin my-helpful-pr`)
-1. Create a new Pull Request
-  - Include links to any relevant github issues.
-  - *Don't* change the VERSION file.
-  - Update `/docs` to include, whenever possible, a new, suitable recommendation about how to use
-    the feature.
-  - Extra Credit: [Confirm it runs and tests pass on the rubies specified in the travis
-    config](.travis.yml). A maintainer will otherwise confirm it runs on these.
-
-1. *Bonus Points* Update [CHANGELOG.md](https://github.com/rails-api/active_model_serializers/blob/master/CHANGELOG.md)
-  with a brief description of any breaking changes, fixes, features, or
-  miscellaneous changes under the proper version section.
-1. Iterate on feedback given by the community (fix syntax, modify bits of code, add
-tests), pushing the new commits to the PR each time
-
-Remember to [squash your commits](CONTRIBUTING.md#about-pull-requests-prs) and rebase off `master`.
-
-#### How maintainers handle pull requests:
-
-- If the tests pass and the pull request looks good, a maintainer will merge it.
-- If the pull request needs to be changed,
-  - you can change it by updating the branch you generated the pull request from
-    - either by adding more commits, or
-    - by force pushing to it
-  - A maintainer can make any changes themselves and manually merge the code in.
-
-#### Commit Messages
-
-- [A Note About Git Commit Messages](http://tbaggery.com/2008/04/19/a-note-about-git-commit-messages.html)
-- [http://stopwritingramblingcommitmessages.com/](http://stopwritingramblingcommitmessages.com/)
-- [ThoughtBot style guide](https://github.com/thoughtbot/guides/tree/master/style#git)
```

Comment:
Maybe a Style section at the end of contributing. How many people do you think might read through them? Or do you want them there so you can quick link to that section during a PR review?